src/components/Tank/Tank.helpers.ts

```diff
@@ -0,0 +1,7 @@
+import { TankColor, TankDirection, TankType } from '../../models/TankObject';
+
+export const tankDirectionsList: TankDirection[] = ['up', 'left', 'down', 'right'];
+export const tankColorsList: TankColor[] = ['yellow', 'white', 'green', 'red'];
+// eslint-disable-next-line max-len
+export const tankTypesList: TankType[] = ['tank-type-1', 'tank-type-2', 'tank-type-3', 'tank-type-4', 'tank-type-5', 'tank-type-6', 'tank-type-7', 'tank-type-8'];
+
```

Comment:
OK. About this. I think we can remove all this and instead:
- [ ] have tankTypesList in the same place as all the items that are going to be removed(BattleCity).
- [ ] Instead of tank colors, you can have 2 properties in the model: relation and hasBonus. The relation should be one of the following values: `self`, `friend`, and `enemy`. If it's self, then it's yellow; if it's friend, then it's green; if it's enemy, then it's white if hasBonus is false or undefined, and red if hasBonus is true.
- [ ] move direction to the BattleCity as well.